package.json

```diff
@@ -18,6 +19,7 @@
     "lint": "jshint js preprocessing validation",
     "jasmine-tests": "jasmine",
     "test": "npm run lint && npm run jasmine-tests && npm run validate",
-    "validate": "node validation/markets-validator.js"
+    "validate": "node validation/markets-validator.js",
+    "start": "http-server -f index.html"
```

Comment:
I think I did that here: https://github.com/wo-ist-markt/wo-ist-markt.github.io/pull/216/commits/1580040cf096aabdc36a0559c633b7b0f1e37853 Would you not agree?